src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -652,13 +634,21 @@ private JsonObject createSemanticWebUnconnectedTitle(String precedingSucceedingT
           .put("value", "0262012103")));
   }
 
-  private void assertPrecedingSucceedingTitles(JsonObject actualPrecedingTitle,
-    JsonObject expected, String precedingInstanceId, String succeedingInstanceId) {
+  private void assertPrecedingTitles(JsonObject actualPrecedingTitle,
+                                     JsonObject expected, String precedingInstanceId) {
```

Comment:
Please use the standard line continuation indentation rather than horizontal alignment, which takes up more horizontal space and is harder to maintain